src/hooks/useAuthorizationRoles/useAuthorizationRoles.js

```diff
@@ -0,0 +1,40 @@
+import { useQuery } from 'react-query';
+
+import { useNamespace, useStripes } from '@folio/stripes/core';
+
+import { useEffect, useState } from 'react';
+import { useTenantKy } from '../useTenantKy';
+
+const ROLES_ENDPOINT = (limit) => `roles?limit=${limit}&query=cql.allRecords=1 sortby name`;
+
+export const useAuthorizationRoles = (tenantId) => {
+  const ky = useTenantKy({ tenantId });
```

Comment:
You can use `useOkapiKy` from `stripes-core`, it supports `tenant` argument now

Reply:
Good to know! Fixed